lib/index.js

```diff
@@ -20,34 +28,55 @@ module.exports = function koala(options) {
   App(app);
 
   // handlers
-  app.errorHandler = require('koa-error')(options.error);
   app.pageNotFoundHandler = pageNotFoundHandler;
 
   // properties
   app.jsonStrict = options.jsonStrict !== false;
 
   // proto stuff
-  require('koa-csrf')(app);
-  require('koa-trace')(app);
-  require('koa-body-parsers')(app);
+  app.use(conditional());
+  app.use(new CSRF({
+    invalidSessionSecretMessage: 'Invalid session secret',
+    invalidSessionSecretStatusCode: 403,
+    invalidTokenMessage: 'Invalid CSRF token',
+    invalidTokenStatusCode: 403,
+    excludedMethods: [ 'GET', 'HEAD', 'OPTIONS' ],
+    disableQuery: false
+  }));
+  app.use(etag());
+  app.use(bodyParser());
+  // app.use(convert(trace));
+  const pug = new Pug({
+    debug: false,
+    pretty: false,
+    compileDebug: false,
+    locals: {},
+    basedir: `${__dirname}/templates`,
```

Comment:
note that this means that _client_ templates are not resolvable.

Reply:
Does this limit pug templates rendered on the server or browser?